### PR TITLE
Fix a bug to extract build_id from xml in freshmaker scraper

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

File: scrapers/freshmaker.py [hunk 1]
```diff
@@ -80,9 +80,9 @@ def query_api_and_update_neo4j(self):
                     # TODO: Change this if a task can trigger multiple builds
                     build_id = xml_root.find(".//*[name='koji_builds'].//string")
 
-                    if build_id:
+                    if isinstance(build_id, ET.Element):
```
Comment (mprahl) [inline scrapers/freshmaker.py]: What are the alternative types that `build_id` could be?

Comment (yashvardhannanavati) [reply scrapers/freshmaker.py]: It can be None, and I tried using `if build_id` but it doesn't work for some reason.

Comment (mprahl) [reply scrapers/freshmaker.py]: That's odd. How about this syntax then?
```python
try:
    build_id = xml_root.find(".//*[name='koji_builds'].//string").text
except AttributeError:
    build_id = None
```

Comment (yashvardhannanavati) [reply scrapers/freshmaker.py]: It doesn't throw an attribute error when the element is not present. It returns None

Comment (mprahl) [reply scrapers/freshmaker.py]: It would when you try to access `.text` on a value that doesn't have like `None`.

Comment (yashvardhannanavati) [reply scrapers/freshmaker.py]: Ah, cool. I see. Thanks

**Comment by mprahl** [comment]
:+1:

**Pull request merged**